Userland/Libraries/LibWeb/WebIDL/DOMException.cpp

```diff
@@ -38,3 +39,35 @@ void DOMException::initialize(JS::Realm& realm)
 }
 
 }
+
+// https://webidl.spec.whatwg.org/#ref-for-serialization-steps%E2%91%A2
+Web::WebIDL::ExceptionOr<void> Web::WebIDL::DOMException::serialization_steps(HTML::SerializationRecord& record, bool, HTML::SerializationMemory&)
```

Comment:
This definition should be within the opened `Web::WebIDL` namespace, rather than outside of it with the fully qualified name in the function name. and return type.